### PR TITLE
setup css global variables

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- The project was missing any type of formating.
How:
- Added common global CSS variables, colors and typography sizes to use throughout different components and blocks.

### CODE REVIEW

**Comment by rccsousa** [comment]
@davelange my plan is to go from top to bottom. starting by defining the globals in here, then i'll branch out for each subsequent section, starting from the header.
Spacers / gutters will be defined as components.
I assume padding and other things will be scoped locally, that's what makes the most sense to me based on what i've been researching.

File: src/app/_css/globals.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  /* Typography */
+  --colfax-medium: "Colfax Medium", Roboto, sans-serif;
+  --colfax-regular: Colfax, Roboto, sans-serif;
+  /* TODO: Replace placeholder TNR for actual Acta*/
+  --acta-book: "Acta Book", "Times New Roman", serif;
+
+
+  h1 {
+    font-size: 56px;
```
Comment (davelange) [inline src/app/_css/globals.css:37]: This approach is valid, though it means you will need to always use the correct html elements to get the intended font sizes. Usually we decouple this, and create something like
```
`--font-size-56`: 56px

h1 {
  font-size: var(--font-size-56)
}
```
Then we can use the --font-size-56 variable anywhere else if needed.

**Review by davelange — APPROVED** [review]

**Comment by davelange** [comment]
> Spacers / gutters will be defined as components.
I assume padding and other things will be scoped locally, that's what makes the most sense to me based on what i've been researching.

This is kind of a rule of thumb, but:
*Padding* should be locally defined in each component, as it represents space within the component boundary
*Spacing between components (margin, gutters)* are not the responsibility of each component, but of the "composer" or layout component. So for a row with gutters:

```
<div class"layout"> 
 <ComponentA />
 <ComponentB />
<div>
```

It should be the .layout styles that define the positioning and gutters of it's children (for example with display: flex, gap: 20px). And if it's a common thing, maybe make it into a <Row> component that receives children, and optionally some parameters for defining spacings etc. Spacer components are also useful, depending on the layout.

**Comment by rccsousa** [comment]
thank you for all the clarifications and insight. i'll put these to use in the comming PRs

**Pull request merged**